test/integration/scenarios/stress/0.transfer.js

```diff
@@ -47,43 +47,33 @@ module.exports = function(params) {
 		}
 
 		describe('sending 1000 bundled transfers to random addresses', () => {
-			var count = 1;
+			before(() => {
+				_.range(maximum).map(() => {
+					var transaction = lisk.transaction.transfer({
+						amount: randomUtil.number(100000000, 1000000000),
+						passphrase: accountFixtures.genesis.password,
+						recipientId: randomUtil.account().address,
+					});
+					transactions.push(transaction);
+				});
 
-			before(done => {
-				async.doUntil(
-					next => {
-						var bundled = [];
-						for (
-							var i = 0;
-							i < params.configurations[0].broadcasts.releaseLimit;
-							i++
-						) {
-							var transaction = lisk.transaction.transfer({
-								amount: randomUtil.number(100000000, 1000000000),
-								passphrase: accountFixtures.genesis.password,
-								recipientId: randomUtil.account().address,
-							});
-							transactions.push(transaction);
-							bundled.push(transaction);
-							count++;
-						}
-						sendTransactionsPromise(bundled).then(next);
-					},
-					() => {
-						return count >= maximum;
-					},
-					() => {
-						done();
-					}
-				);
+				const limit = params.configurations[0].broadcasts.releaseLimit;
+
+				return _.chunk(transactions, limit).map(bundledTransactions => {
+					return sendTransactionsPromise(bundledTransactions);
```

Comment:
Will this really work? as in the end each transaction will be posted as individual transaction.

Reply:
Yes, this will bundle 25 transactions and send it as one single transaction. The `sendTransactionsPromise` basically accepts an array of a transaction when we want to post a single transaction we do `sendTransactionsPromise([transaction])` in this case `bundledTransactions` is an array of 25 transactions.

Reply:
https://github.com/LiskHQ/lisk/pull/2024#pullrequestreview-122154869